dell-csi-helm-installer/csi-offline-bundle.md

```diff
@@ -78,13 +78,13 @@ For example, here is the output of a request to build an offline bundle for the
 *
 * Pulling and saving container images
 
-   quay.io/dell/container-storage-modules/csi-isilon:v2.14.0
+   quay.io/dell/container-storage-modules/csi-isilon:v2.15.0
    quay.io/dell/container-storage-modules/csi-metadata-retriever:v1.10.0
    quay.io/dell/container-storage-modules/csipowermax-reverseproxy:v2.13.0
-   quay.io/dell/container-storage-modules/csi-powermax:v2.14.0
-   quay.io/dell/container-storage-modules/csi-powerstore:v2.14.0
-   quay.io/dell/container-storage-modules/csi-unity:v2.14.0
-   quay.io/dell/container-storage-modules/csi-vxflexos:v2.14.0
+   quay.io/dell/container-storage-modules/csi-powermax:v2.15.0
+   quay.io/dell/container-storage-modules/csi-powerstore:v2.15.0
+   quay.io/dell/container-storage-modules/csi-unity:v2.15.0
+   quay.io/dell/container-storage-modules/csi-vxflexos:v2.15.0
    quay.io/dell/container-storage-modules/csm-authorization-sidecar:v1.14.0
    quay.io/dell/container-storage-modules/csm-metrics-powerflex:v1.12.0
    quay.io/dell/container-storage-modules/csm-metrics-powerscale:v1.9.0
```

Comment:
Not a comment for the change. But on line 97 I see `nginxinc/nginx-unprivileged:1.27`. Do we want to change this one to quay as well `quay.io/nginx/nginx-unprivileged:1.27` to keep consistency with operator?

Reply:
This offline bundle uses the helm chart which uses nginxinc/nginx-unprivileged:1.27, not quay.io. Although, we could change that?